lib/services/identity.go

```diff
@@ -267,12 +267,6 @@ type Identity interface {
 	// CreateUserRecoveryAttempt logs user recovery attempt.
 	CreateUserRecoveryAttempt(ctx context.Context, user string, attempt *types.RecoveryAttempt) error
 
-	// GetUserRecoveryAttempts returns user recovery attempts sorted by oldest to latest time.
-	GetUserRecoveryAttempts(ctx context.Context, user string) ([]*types.RecoveryAttempt, error)
```

Comment:
How does one read recovery attempts, as a Teleport user/admin?

Are there leftover methods, meant to implement this interface, that we could now remove?

Reply:
You bring up a good point, maybe I should bring this API back?

Reply:
I thought we wanted the attempts tracked somewhere, but maybe they are already in audit? I don't recall from the top of my head how this plugs into the wider system.

Reply:
We do log failures here: https://github.com/gravitational/teleport/pull/35325/files#diff-58ee6a3f9df8cbca83069a64e445a0b91c16352c77a92f8cdec8be15847c7d89L362

However I am not sure if we have another need to lookup current failures on an account?